lib/posix-process/clone.c

```diff
@@ -515,15 +515,6 @@ int uk_clone(struct clone_args *cl_args, size_t cl_args_len,
 		    (void *) child->tlsp,
 		    (child->tlsp != child->uktlsp) ? "custom" : "Unikraft");
 
-	if (!(cl_args->flags & CLONE_THREAD)) {
-		uk_pr_debug("Creating new process for child\n");
```

Comment:
In commit message of ` lib/posix-process: Restrict pthread creation to clone() `: `s/rename then/rename them/`

Reply:
This commit is now gone.